docs/reference/mapping/params/dynamic.asciidoc

```diff
@@ -42,7 +42,7 @@ GET my-index-000001/_mapping
 [[dynamic-inner-objects]]
 ==== Setting `dynamic` on inner objects
 <<object,Inner objects>> inherit the `dynamic` setting from their parent
-object or from the mapping type. In the following example, dynamic mapping is
+object. In the following example, dynamic mapping is
 disabled at the type level, so no new top-level fields will be added
```

Comment:
Maybe we should also not mention "type" here then. Maybe "root" level or "root mapping" level is more accurate now? Same goes for the mentions of "type" in the callouts after the example.